src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -38,6 +38,12 @@
   "minecolonies.config.badvisitorchance": "Bad recruitment chance",
   "minecolonies.config.badvisitorchance.comment": "Chance to get an evil tavern visitor which can run off with the resources, in % [Default: 2]",
 
+  "minecolonies.config.researchspeedmultiplier": "Research Speed Multiplier",
+  "minecolonies.config.researchspeedmultiplier.comment": "Rate research is completed.  Default will correspond to ~ 1h, 2h, 4h, 8h, 16h, 32h. \n Because of sleep cycles, high values will progress much faster. \n Generally only recommended for packs that increase research unlock costs, or add many additional researches.",
```

Comment:
Shadow, doesn't "the" need to be capitalized at the beginning of the second sentence? 

Also, the "Because of sleep cycles" is confusing and not explained. Just saying "High values will progress much faster." would be fine in my opinion.

Should it be "corresponds to ~1h, 2h, 4h" with the tilde moved closer?